eos/form-factors/parametric-kkrvd2024.cc

```diff
@@ -0,0 +1,389 @@
+    }
+
+    double
+    KKRvD2024FormFactors<VacuumToPiPi>::_b0_fp_I1(const double & chi, const complex<double> & zr) const
+    {
+        const complex<double> z0 = this->z(0.0);
+
+        const complex<double> phitilde_z0      = this->phitilde_p(z0, chi);
+        const complex<double> phitilde_m1      = this->phitilde_p(-1.0, chi);
+        const complex<double> phitildeprime_m1 = this->phitildeprime_p(-1.0, chi);
+
+        const complex<double> x_z0      = 1.0 / (phitilde_z0 * std::norm(z0 - zr));
+        const complex<double> x_m1      = 1.0 / (phitilde_m1 * std::norm(1.0 + zr));
+        const complex<double> xprime_m1 = (2.0 * (1 + std::real(zr)) * phitilde_m1 - std::norm(1.0 + zr) * phitildeprime_m1 ) / power_of<2>(std::norm(1.0 + zr) * phitilde_m1);
+
+        std::array<double, 10> b;
+        b[0] = 0.0;
+        b[1] = 0.0;
+        std::copy(_b_fp_I1.cbegin(), _b_fp_I1.cend(), b.begin()+2);
+
+        complex<double> sum = 0.0;
+        for (auto i = 0u; i < b.size(); i++)
+        {
+            sum += b[i] * ( (pow(-1, i) * i + pow(z0, i-1)) * x_m1 - (pow(-1, i) + pow(z0, i-1)) * xprime_m1 );
+        };
+
+        const double b0 = std::real((x_m1 - xprime_m1 - x_z0 * z0 * sum) / (x_z0 * (x_m1 - (1.0 + z0) * xprime_m1)));
+        return b0;
+    }
+
+    double
+    KKRvD2024FormFactors<VacuumToPiPi>::_b1_fp_I1(const double & chi, const complex<double> & zr) const
+    {
+        const complex<double> z0 = this->z(0.0);
+
+        const complex<double> phitilde_z0      = this->phitilde_p(z0, chi);
+        const complex<double> phitilde_m1      = this->phitilde_p(-1.0, chi);
+        const complex<double> phitildeprime_m1 = this->phitildeprime_p(-1.0, chi);
+
+        const complex<double> x_z0      = 1.0 / (phitilde_z0 * std::norm(z0 - zr));
+        const complex<double> x_m1      = 1.0 / (phitilde_m1 * std::norm(1.0 + zr));
+        const complex<double> xprime_m1 = (2.0 * (1 + std::real(zr)) * phitilde_m1 - std::norm(1.0 + zr) * phitildeprime_m1 ) / power_of<2>(std::norm(1.0 + zr) * phitilde_m1);
+
+        std::array<double, 10> b;
+        b[0] = 0.0;
+        b[1] = 0.0;
+        std::copy(_b_fp_I1.cbegin(), _b_fp_I1.cend(), b.begin()+2);
+
+        complex<double> sum = 0.0;
+        for (auto i = 0u; i < b.size(); i++)
+        {
+            sum += b[i] * (pow(-1, i) * i * x_m1 + (pow(-1, i+1) + pow(z0, i)) * xprime_m1);
+        };
+
+        const double b1 = std::real((sum * x_z0 -  xprime_m1) / ( x_z0 * (x_m1 - (1.0 + z0) * xprime_m1)));
+        return b1;
+    }
+
+    complex<double>
+    KKRvD2024FormFactors<VacuumToPiPi>::f_p(const double & q2) const
+    {
+        static const double eps = 1.0e-12;
+        return f_p(complex<double>(q2, eps));
+    }
+
+    complex<double>
+    KKRvD2024FormFactors<VacuumToPiPi>::f_p(const complex<double> & q2) const
+    {
+        const auto z        = this->z(q2);
+        const auto chi      = 0.0258815; // GeV^-2, from Meril at Q^2 = 1 GeV^2
```

Comment:
Can we make this an EOS parameter?

Reply:
Would leave it for a future modification (i.e., full e.m. FF analysis).

Reply:
At least the comment should be modified

Reply:
Yes. This is based on the analytical formula in Buck/Lebed, correct?

Reply:
Yes! Which might benefit from a cross check :)